weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1327,8 +1325,8 @@ def _ch_call_to_call_schema(ch_call: SelectableCHCallSchema) -> tsi.CallSchema:
         op_name=ch_call.op_name,
         started_at=_ensure_datetimes_have_tz(ch_call.started_at),
         ended_at=_ensure_datetimes_have_tz(ch_call.ended_at),
-        attributes=_dict_dump_to_dict(ch_call.attributes_dump),
-        inputs=_dict_dump_to_dict(ch_call.inputs_dump),
+        attributes=_nullable_any_dump_to_any(ch_call.attributes_dump or "{}"),
+        inputs=_nullable_any_dump_to_any(ch_call.inputs_dump or "{}"),
```

Comment:
I would recommend modifying `SelectableCHCallSchema` to correctly type the response payload so that this would be caught by typer